### PR TITLE
[WIP Y2K-1122] try to bust the usercard cache

### PR DESCRIPTION
**Pull request by jzila**

### CODE REVIEW

File: go/service/user_handler.go [hunk 1]
```diff
@@ -114,6 +114,7 @@ func (r *userHandler) userBlocked(m libkb.MetaContext, cli gregor1.IncomingInter
 			m.Warning("Error handling UserBlocked message: %s", tmp)
 		}
 	}
+	r.G().CardCache().Delete(msg.Uid)
```
Comment (patrickxb) [inline go/service/user_handler.go]: don't you want to delete badUIDs?

Comment (maxtaco) [reply go/service/user_handler.go]: @jzila, @patrickxb is correct

Comment (patrickxb) [reply go/service/user_handler.go]: I think this PR will work: https://github.com/keybase/client/pull/21430

If there's anything that it is missing that is in this one, let me know.

**Comment by jzila** [comment]
Superseded by #21430

**Pull request closed without merging**